src/ContactInformation/ContactInfoViewGroup/AddressInfoView.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
 import { Row, Col } from '@folio/stripes/components';
 import { AddressView } from '@folio/stripes/smart-components';
 import css from '../ContactInformationView.css';
+import CatIDToLabel from '../../Utils/CatIDToLabel';
```

Comment:
just a nit: `CatIDToLabel` is a function, I think it's a common style to make function names starting from small letter, like `catIDToLabel`